apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -73,13 +83,16 @@ export function InstructorAiGenerateDraftEditor({
                   >
                     <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
                     <input type="hidden" name="__action" value="regenerate_question" />
-                    <div class="form-group">
-                      <label for="user-prompt-llm">What needs to be changed?</label>
-                      <textarea name="prompt" id="user-prompt-llm" class="form-control"></textarea>
-                    </div>
-                    <button class="btn btn-primary">
+                    <textarea
+                      name="prompt"
+                      id="user-prompt-llm"
+                      class="form-control mb-2"
+                      placeholder="What would you like to change?"
+                      aria-label="Modification instructions"
+                    ></textarea>
```

Comment:
Can we make it so that pressing Enter in this textarea will submit the form? That's the UX I'm used to from chatbots. Preferably with shift-enter to get an actual newline, if that's possible?

Reply:
Done in 4ba8274. TIL about the `requestSubmit()` method on forms!

Reply:
TIL also! Very cool.